safe_transaction_service/history/tests/test_helpers.py

```diff
@@ -0,0 +1,43 @@
+from unittest.mock import MagicMock
+
+from django.test import TestCase
+
+from hexbytes import HexBytes
+
+from gnosis.safe.tests.safe_test_case import SafeTestCaseMixin
+
+from safe_transaction_service.history.helpers import DelegateSignatureHelperV2
+
+
+class TestDelegateSignatureHelperV2(SafeTestCaseMixin, TestCase):
+
+    def test_calculate_hash(self):
+        # Mock calculate_totp
+        DelegateSignatureHelperV2.calculate_totp = MagicMock(
+            side_effect=lambda previous: 123456 if not previous else 654321
+        )
+
+        delegate_address = "0x1234567890123456789012345678901234567890"
+        chain_id = 1
+
+        expected_hash_previous_false = HexBytes(
```

Comment:
It takes some time to me to understand what means the false, previous_false did me to think that was a previous wrong hash. 
I think that could help add a comment to this line as `# Hash calculated when totp previous is false` the same for the previous true.

Reply:
Added.